modules/cluster-logging-collector-log-forward-es.adoc

```diff
@@ -65,8 +66,10 @@ spec:
 <8> Specify which log types should be forwarded using that pipeline: `application,` `infrastructure`, or `audit`.
 <9> Specify the output to use with that pipeline for forwarding the logs.
 <10> Optional: Specify the `default` output to send the logs to the internal Elasticsearch instance.
-<11> Optional: One or more labels to add to the logs.
-<12> Optional: Configure multiple outputs to forward logs to other external log aggregtors of any supported type:
+<11> Optional: Forward structured JSON log entries as JSON objects in the `structured` field. The log entry must contain valid structured JSON; otherwise, the `structured` field will be missing or empty.
+// TBD
```

Comment:
@alanconway What should we say about using this capability with external ES?

Reply:
I've created an enhancement request to cover this https://issues.redhat.com/browse/RHDEVDOCS-3200